_rules/SC1-2-1-video-only-element-transcript.md

```diff
@@ -0,0 +1,125 @@
+---
+name: video only element has transcript
+description: |
+  Non-streaming `video` elements without audio must have all visual information available in a transcript.
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- Audio output
+- Visual output
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible](#visible) where the video doesn't contain audio.
+
+### Expectation
+
+The visual information of each test target is available through a text transcript that is available either on the page or through a link. The text transcript needs to be [visibile](#visible) and [included-in-the-accessibility-tree](#included-in-the-accessibility-tree).
```

Comment:
Also in this one, there should be two more failed test cases: one for the transcript not being visible and another for the transcript not being included in the accessibility tree.

Reply:
Awesome suggestion! Added!